onnxruntime/core/optimizer/attention_fusion.cc

```diff
@@ -99,9 +99,10 @@ static NodeArg& MergeQkvWeights(Graph& graph, int64_t hidden_size,
   assert(nullptr != q_tensor);
   assert(nullptr != k_tensor);
   assert(nullptr != v_tensor);
-  Initializer q_initializer(*q_tensor, graph.ModelPath());
-  Initializer k_initializer(*k_tensor, graph.ModelPath());
-  Initializer v_initializer(*v_tensor, graph.ModelPath());
+  InitializerOption opt{graph.ModelPath(), graph.ExternalDataMap()};
```

Comment:
>graph.ModelPath(), graph.ExternalDataMap()}; [](http://example.com/codeflow?start=24&length=44)

My understanding that loading and resolving the initializes should take place before optimization layer kicks in and all the initializers data should already be within the graph?

Reply:
I actually take that back.